server/util/redact/redact.go

```diff
@@ -206,6 +207,41 @@ func redactCmdLine(tokens []string) {
 	stripExplicitCommandLineFromCmdLine(tokens)
 }
 
+func RedactTxt(txt string) string {
```

Comment:
nit: avoid abbreviation since it's an exported func

```suggestion
func RedactText(txt string) string {
```